src/js/main.js

```diff
@@ -0,0 +1,62 @@
+import Mouse from './mouse';
+import SelectService from './select.service';
+import * as PolygonService from './polygon.service';
+
+window.onload = () => {
+    const canvas = document.getElementById("canvas");
```

Comment:
It's nice to create class smth like `App` and onload start it. smth like
```js
window.onload = () => {
    const canvas = document.getElementById("canvas");
    const app = new App(canvas);
```
And all other lines move to app constructor